k8s/demo/common/divorce/div-demo.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-demo
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.div-fps: glob:prod-*
+    flux.weave.works/tag.div-hm: glob:prod-*
+    flux.weave.works/tag.div-demo-fees: glob:prod-*
+spec:
+  releaseName: div-demo
+  timeout: 900
```

Comment:
is this default value?

Reply:
I guess so, copied from https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/common/money-claims/cmc-demo.yaml#L28
We can tweak later if it doesn't work for us